padertorch/train/optimizer.py

```diff
@@ -37,9 +38,11 @@ def clip_grad(self):
         # Todo: report clipped and unclipped
         # Todo: allow clip=None but still report grad_norm
         grad_clips = self.gradient_clipping
-        return torch.nn.utils.clip_grad_norm_(
-            self.parameters, grad_clips
-        )
+        if isinstance(self.parameters[0], dict) and 'params' in self.parameters[0]:
+            params = itertools.chain(*[param_group['params'] for param_group in self.parameters])
```

Comment:
Is that a bug in PyTorch, that the optimizers support more inputs than the `clip_grad_norm_`?
Could you add a comment here, which special case is solved?